ansible/roles/operator-pipeline/templates/openshift/tasks/github-pipelinerun-summary.yml

```diff
@@ -128,7 +128,7 @@ spec:
 
         if [[ "$ORGANIZATION" == "community-operators" ]]
         then
-          DOC_LINK="https://redhat-openshift-ecosystem.github.io/operator-pipelines/users/contributing-prerequisites/"
+          DOC_LINK="https://raw.githubusercontent.com/redhat-openshift-ecosystem/operator-pipelines/main/docs/community-operators-troubleshooting.md/"
```

Comment:
Please use a link to our official user-facing documentation page: https://redhat-openshift-ecosystem.github.io/operator-pipelines/ and make the `[community-operators-troubleshooting.md](https://github.com/redhat-openshift-ecosystem/operator-pipelines/pull/760/files#diff-6e349552b5191800846bf0bab1baf4428cfaa42077061e913fffac3ae364480d)` available there.